content/en/docs/devops-user-guide/how-to-use/use-pipeline-templates.md

```diff
@@ -0,0 +1,280 @@
+
+   | Credential ID   | Type                | Where to use |
+| --------------- | ------------------- | ------------ |
+   | dockerhub-id    | Account Credentials | Docker Hub   |
+| demo-kubeconfig | kubeconfig          | Kubernetes   |
+   
+2. In total, you have two credentials in the list.
+
+   ![two-credentials](/images/docs/devops-user-guide/using-devops/use-pipeline-templates/two-credentials.png)
+
+3. Log out and log back in as `project-admin`. You need to create two projects `kubesphere-sample-dev` and `kubesphere-sample-prod` to use them as the development environment and production environment respectively. Make sure you invite the account `project-regular` to these two projects with the role of `operator`.
+
+### Step 2: Create a pipeline 
+
+1. Go to your DevOps project, and then click **Create** in **Pipelines**.
+2. In the dialog that appears, name it `cicd-pipeline` and click **Next**.
+
+   ![cicd-pipeline](/images/docs/devops-user-guide/using-devops/use-pipeline-templates/cicd-pipeline.png)
+
+3. On the **Advanced Settings** tab, click **Add Parameter** to add three string parameters as follows. These parameters will be used in the Docker command of the pipeline. Click **Create** when you finish adding.
+
+   | Parameter Type | Name                | Value         | Description                                                  |
+   | -------------- | ------------------- | ------------- | ------------------------------------------------------------ |
+   | String         | REGISTRY            | `docker.io`   | This is the image registry address. This example uses `docker.io`. |
+   | String         | DOCKERHUB_NAMESPACE | Docker ID     | You Docker Hub account or the organization name under the account. |
+   | String         | APP_NAME            | `cicd-sample` | The app name.                                                |
+
+4. The pipeline created will appear in the list.
+
+   ![cicd-pipeline-created](/images/docs/devops-user-guide/using-devops/use-pipeline-templates/cicd-pipeline-created.png)
+
+### Step 3: Edit the pipeline
+
+Click the pipeline to go to its detail page, and then click **Edit Pipeline** under the **Pipeline** tab. In the dialog that appears, click the pipeline template for continuous integration & delivery.
+
+![click-cicd-template](/images/docs/devops-user-guide/using-devops/use-pipeline-templates/click-cicd-template.png)
+
+In the graphical editing panel, there are six predefined stages with built-in steps. Likewise, you can customize these steps based on your needs. For the **clone code** stage and the **build & push** stage, you can follow [the same operations as described above for CI pipeline](#step-3-edit-the-pipeline).
+
+#### Stage 4: push latest
+
+1. Click push latest to select this stage, and then click the wrench icon on the right of the first **sh** step.
```

Comment:
Click push latest to select
->
Click `push latest` to select

Reply:
I'll make it bold here.